pkg/crane/catalog.go

```diff
@@ -29,5 +29,5 @@ func Catalog(src string, opt ...Option) (res []string, err error) {
 		return nil, err
 	}
 
-	return remote.Catalog(context.TODO(), reg, o.remote...)
+	return remote.Catalog(context.Background(), reg, o.remote...)
```

Comment:
Not plumbing context through here?

Reply:
The `remote.WithContext` in `o.remote...` will override it.